Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/repository/storage/VaccinationContainer.kt

```diff
@@ -23,43 +20,29 @@ import org.joda.time.Instant
 import org.joda.time.LocalDate
 import java.util.Locale
 
-@Keep
-data class VaccinationContainer internal constructor(
-    @SerializedName("vaccinationQrCode") val vaccinationQrCode: QrCodeString,
-    @SerializedName("scannedAt") val scannedAt: Instant,
-    @SerializedName("notifiedExpiresSoonAt") val notifiedExpiresSoonAt: Instant? = null,
-    @SerializedName("notifiedExpiredAt") val notifiedExpiredAt: Instant? = null,
-    @SerializedName("notifiedInvalidAt") val notifiedInvalidAt: Instant? = null,
-    @SerializedName("notifiedBlockedAt") val notifiedBlockedAt: Instant? = null,
-    @SerializedName("lastSeenStateChange") val lastSeenStateChange: State? = null,
-    @SerializedName("lastSeenStateChangeAt") val lastSeenStateChangeAt: Instant? = null,
-    @SerializedName("certificateSeenByUser") val certificateSeenByUser: Boolean = true,
-    @SerializedName("recycledAt") override val recycledAt: Instant? = null,
+data class VaccinationContainer(
+    internal val data: StoredVaccinationCertificateData,
```

Comment:
Could you also rename to VaccinationCertificateContainer to align with the other containers?

Reply:
updated